server/base/src/main/java/org/apache/accumulo/server/master/balancer/HostRegexTableLoadBalancer.java

```diff
@@ -56,18 +61,24 @@
  * <b>table.custom.balancer.host.regex.pool.check=5m</b><br>
  * Regex matching can be based on either the host name (default) or host ip address. To set this balancer to match the regular expressions to the tablet server
  * IP address, then set the following property:<br>
- * <b>table.custom.balancer.host.regex.is.ip=true</b>
+ * <b>table.custom.balancer.host.regex.is.ip=true</b><br>
+ * It's possible that this balancer may create a lot of migrations. To limit the number of migrations that are created during a balance call, set the following
+ * property (default 250):<br>
+ * <b>table.custom.balancer.host.regex.concurrent.migrations</b>
  *
  */
-public class HostRegexTableLoadBalancer extends TableLoadBalancer {
+public class HostRegexTableLoadBalancer extends TableLoadBalancer implements ConfigurationObserver {
 
   private static final Logger LOG = LoggerFactory.getLogger(HostRegexTableLoadBalancer.class);
   public static final String HOST_BALANCER_PREFIX = Property.TABLE_ARBITRARY_PROP_PREFIX.getKey() + "balancer.host.regex.";
-  public static final String HOST_BALANCER_OOB_CHECK = Property.TABLE_ARBITRARY_PROP_PREFIX.getKey() + "balancer.host.regex.oob.period";
+  public static final String HOST_BALANCER_OOB_CHECK_KEY = Property.TABLE_ARBITRARY_PROP_PREFIX.getKey() + "balancer.host.regex.oob.period";
```

Comment:
How many oobs does a host balancer have to check?  (in other words a more obvious property name might be nice).